dev-qt/qtwebengine/qtwebengine-5.15.2.9999.ebuild

```diff
@@ -55,7 +55,7 @@ RDEPEND="
 	media-libs/libpng:0=
 	>=media-libs/libvpx-1.5:=[svc(+)]
 	media-libs/libwebp:=
-	media-libs/mesa[egl,X(+)]
+	media-libs/mesa[egl(+),X(+)]
```

Comment:
I suspect this should be changed to libglvnd, indeed.

Reply:
does it need that dependency at all though? at least it didn't come up in lddtree nor the preserved-libs output; @Chiitoo?

Reply:
Hmmm, I'm not sure.

It does build with a mesa without egl... that's all I can say about that at his time.

Reply:
Scimming through the qtwebengine build system will be hopeless, but here is the dependency from www-client/chromium where it is coming from I guess:
`media-libs/mesa:=[gbm(+)]`

Reply:
`dev-qt/qtwebengine-5.15.2_p20210824-r1` was built successfully completely without `media-libs/mesa` here, and since there is nothing to be seen with `lddtree` either, I have determined that it uses `media-libs/libglvnd` **headers only**.

Without `media-libs/libglvnd`, it will print this warning:
`WARNING: Khronos development headers required to build QtWebEngine are missing (see mesa/libegl1-mesa-dev)`

Funny enough that it will not fail to configure but happily proceed, but we know that already and make it fail in `src_install() `instead, checking for existence of the main .so file.